### DISCUSSION TITLE
I can't get the bio to update?  All other author params work.

### DISCUSSION THREAD

**Discussion by wzrdk3lly** [jpanther/congo#38] [Q&A] [answered]
First and foremost thanks jpanther for creating this theme. It's helped me learn how to use github and understand how a site is built. 

My Question: I have inserted the author parameters 
```
[author]
  name = "Wzrdk3lly"
  image = "img/unicorn.png"
  bio = "Just a techie that likes to hack, collect NFTs, and skate."
```
![image](https://user-images.githubusercontent.com/40729327/142972836-64f1d9a7-09f5-483a-b494-48e3ceea3d5a.png)

Everything appears to work except for the bio. I don't understand what I'm doing wrong. Can someone help me please?

**Comment by jpanther** [comment, marked as answer]
Thanks @mkduke424, the issue here is due to the fact that the `bio` parameter serves a different purpose. It controls the text that is inserted in the author footer below each article...
![Screen Shot 2021-11-23 at 16 03 45](https://user-images.githubusercontent.com/4462786/142973942-fae47ada-be37-4976-9886-f30d70615ff4.JPG)

What I think you're looking for is the homepage content. You can simply place this in a `content/_index.md` file and it will be placed on the homepage. You can use any standard Markdown or Hugo shortcodes in this file.

**Comment by wzrdk3lly** [comment]
Awesome! Thanks @jpanther. Thanks for the super fast reply too :)
